aries-site/src/pages/templates/forms/index.mdx

```diff
@@ -317,3 +319,16 @@ Use Single-column layouts create clear paths for users. Multi-column layouts can
     <ColumnFormDont />
   </Example>
 </BestPracticeGroup>
+
+## Managing child objects within a form
+
+Applications often have scenarios where an object may have relationships to multiple 
```

Comment:
This first sentence felt abstract to me. Maybe simplifying to:

Applications often have scenarios where an object has multiple child objects, such as:
- Creating an account...

Reply:
done.